tools/setup-musl-build.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+# Usage:
+#   tools/setup-musl-build.sh <build-directory> <options…>
+# E.g.
+#   tools/setup-musl-build.sh build-musl -Dbuildtype=debugoptimized && ninja -C build-musl
+
+set -eu
+
+BUILD_DIR="${1:?}"
+shift
+
+SETUP_DIR="${BUILD_DIR}/extra"
+
+mkdir -p "${SETUP_DIR}/usr/include"
+mkdir -p "${SETUP_DIR}/usr/lib64/pkgconfig"
```

Comment:
Not your fault, but the extra pkgconfig directory is currently not used. Could you drop the line?